MVCDemos/DevExtreme.MVC.Demos.csproj

```diff
@@ -802,6 +802,7 @@
     <Content Include="Content\DemosStyles\Charts\PieWithCustomLabels.css" />
     <Content Include="Content\DemosStyles\Charts\PieWithMultipleSeries.css" />
     <Content Include="Content\DemosStyles\Charts\PieWithSmallValuesGrouped.css" />
+    <Content Include="Content\DemosStyles\Charts\PieWithCustomStyles.css" />
```

Comment:
As far as i remember we should also add a new `png` here.
Please, build the project locally and test that everything is ok